themes/default/content/docs/intro/languages/dotnet.md

```diff
@@ -12,7 +12,9 @@ aliases: ["/dotnet/"]
 
 <img src="/logos/tech/dotnet.png" align="right" width="150" style="padding:8px; margin-top: -64px">
 
-Pulumi supports infrastructure as code using any .NET language. You can use your favorite .NET tools &mdash; such as editors, package managers, build systems, and test frameworks &mdash; to create, deploy, and manage infrastructure on any cloud, including Azure, AWS, and Google Cloud.
+Pulumi supports writing your infrastructure as code in any .NET language programs running on dotnet using any [Supported version](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core#lifecycle).
```

Comment:
```suggestion
Pulumi supports writing your infrastructure as code in any .NET language programs running on dotnet using any [supported version](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core#lifecycle).
```

Here and similarly below: this wouldn't usually be capitalised, and it doesn't look like the term "supported version" is given any special treatment on the linked pages. The word "Support" does appear capitalised in product names (e.g., "Oracle Premier Support"), and when it's the first word in a sentence or heading -- but not in general, and not within the term "supported versions".